File: MPI/mpicheck/tests/functions/2.0/MPI_Comm_get_name.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    char *var_1;
    int *var_2;
    int ret;
    /* calls */
    ret = MPI_Comm_get_name(var_0, var_1, var_2);
    ret = PMPI_Comm_get_name(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Comm_get_name.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       CHARACTER*(10) var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_comm_get_name(var_0, var_1, var_2, var_3)
       call pmpi_comm_get_name(var_0, var_1, var_2, var_3)
        end program main
    